secp256k1-sys/src/lib.rs

```diff
@@ -816,8 +816,7 @@ mod fuzz_dummy {
                     0
                 } else {
                     ptr::copy(input.offset(1), (*pk).0.as_mut_ptr(), 64);
-                    test_cleanup_pk(pk);
```

Comment:
Can we instead just drop this line and keep the `test_pk_validate` and have it work?

Reply:
I had to change how keys are stored and the function `test_cleanup_pk` was basically an empty function. See the comment on the top module for exact implementation,